ISSUE_TRIAGE.md

```diff
@@ -29,6 +29,28 @@ The first step is in determining whether the issue is a bug, help question or fe
 4. A reproduction should be confirmed by at least one person other than the original reporter. Run the reproduction and validate that the bug exists; then make a note of your findings on the issue. If a reproduction is supplied but doesn't work, add the `can't-reproduce` label and make a comment describing what happened.
 5. Finally, once you've confirmed the reproduction add the `confirmed` label and [classify](#classification) the issue (removing the `can't-reproduce` label if it exists).
 
+#### Bug Issue Lifespan
+
+To help keep issues in this repository under control, and make sure the most important problems are visible to maintainers, unresolved issues (lacking recent activity) should be closed after a certain amount of time has elapsed. 
+
+##### Issues Labelled with `pull-requests-encouraged`
+
+- Open `pull-requests-encouraged` issues should be closed after one month of inactivity, unless someone has clearly identified that they are interested in working on the issue.
+- When closing, the `not-implemented` label should be added.
```

Comment:
@benjamn I know we talked about reversing this so completed `pull-requests-encouraged` issues have an `implemented` label added instead (meaning people looking for PR's to work on can do a `label:pull-requests-encouraged -label:implemented` GH search). I think in practice this might prove cumbersome however, since some PR issues are closed automatically when the PR is merged (when using `fixes` for example). In that case we'd have to remember to always go back to the original PR issue and add the `implemented` label. When triaging issues, it's likely easier to add a `not-implemented` label since we're already in triage mode and will be manually closing the issue anyways.

Reply:
When I first suggested using `implemented` instead of `not-implemented`, I was imagining it might be annoying to add the `not-implemented` tag to every `pull-requests-encouraged` issue, but now I see that simply being tagged with `pull-requests-encouraged` and _still open_ implies that the issue has not yet been implemented, so we don't have to add the `not-implemented` tag explicitly. After we _close_ a `pull-requests-encouraged` issue without implementing it, that's when we would add `not-implemented`, which seems fine!